editor/writing-flow/index.js

```diff
@@ -20,12 +35,40 @@ class WritingFlow extends Component {
 
 		this.onKeyDown = this.onKeyDown.bind( this );
 		this.bindContainer = this.bindContainer.bind( this );
+		this.isLastEditable = this.isLastEditable.bind( this );
+		this.isFirstEditable = this.isFirstEditable.bind( this );
```

Comment:
Binding here is unnecessary, as these methods are always explicitly called as instance methods (`this.isLastEditable( args )`) and are never found as standalone function references (e.g. `{ onSomeEvent: this.isLastEditable }`). In any case, I just pushed a small refactor to eliminate them altogether.